Silent-Vampires/Welcome-to-Nashville-Silent-Vampires
# Rc meetup create element

# Description

I re-factored the meetup code to use createElement and appendChild

Fixes # (issue)

## Type of change

Please delete options that are not relevant.



# Testing Instructions

git fetch --all
git checkout rc-meetup-create-element
run http server
confirm that meetup is pulling test results into the results field



# Checklist:
